groups/bsl/bslstl/bslstl_optional.t.cpp

```diff
@@ -42,7 +61,162 @@ using namespace bsl;
+// [31] bool operator>=(const optional<LHS_TYPE>&, const RHS_TYPE&);
+// [31] bool operator==(const optional<LHS_TYPE>&, const optional<RHS_TYPE>&);
+// [31] bool operator!=(const optional<LHS_TYPE>&, const optional<RHS_TYPE>&);
+// [31] bool operator< (const optional<LHS_TYPE>&, const optional<RHS_TYPE>&);
+// [31] bool operator<=(const optional<LHS_TYPE>&, const optional<RHS_TYPE>&);
+// [31] bool operator>=(const optional<LHS_TYPE>&, const optional<RHS_TYPE>&);
+// [31] bool operator> (const optional<LHS_TYPE>&, const optional<RHS_TYPE>&);
+// [31] bool operator==(const nullopt_t&, const optional<RHS_TYPE>&);
+// [31] bool operator!=(const nullopt_t&, const optional<RHS_TYPE>&);
+// [31] bool operator< (const nullopt_t&, const optional<RHS_TYPE>&);
+// [31] bool operator<=(const nullopt_t&, const optional<RHS_TYPE>&);
+// [31] bool operator> (const nullopt_t&, const optional<RHS_TYPE>&);
+// [31] bool operator>=(const nullopt_t&, const optional<RHS_TYPE>&);
+// [31] bool operator==(const LHS_TYPE&, const optional<RHS_TYPE>&);
+// [31] bool operator!=(const LHS_TYPE&, const optional<RHS_TYPE>&);
+// [31] bool operator< (const LHS_TYPE&, const optional<RHS_TYPE>&);
+// [31] bool operator<=(const LHS_TYPE&, const optional<RHS_TYPE>&);
+// [31] bool operator> (const LHS_TYPE&, const optional<RHS_TYPE>&);
+// [31] bool operator>=(const LHS_TYPE&, const optional<RHS_TYPE>&);
+// [31] bool operator==(const std::optional<LHS_TYPE>&,
+//                      const optional<RHS_TYPE>&);
+// [31] bool operator!=(const std::optional<LHS_TYPE>&,
+//                      const optional<RHS_TYPE>&);
+// [31] bool operator< (const std::optional<LHS_TYPE>&,
+//                      const optional<RHS_TYPE>&);
+// [31] bool operator<=(const std::optional<LHS_TYPE>&,
+//                      const optional<RHS_TYPE>&);
+// [31] bool operator>=(const std::optional<LHS_TYPE>&,
+//                      const optional<RHS_TYPE>&);
+// [31] bool operator> (const std::optional<LHS_TYPE>&,
+//                      const optional<RHS_TYPE>&);
+// [31] bool operator==(const optional<LHS_TYPE>&,
+//                      const std::optional<RHS_TYPE>&);
+// [31] bool operator!=(const optional<LHS_TYPE>&,
+//                      const std::optional<RHS_TYPE>&);
+// [31] bool operator< (const optional<LHS_TYPE>&,
+//                      const std::optional<RHS_TYPE>&);
+// [31] bool operator<=(const optional<LHS_TYPE>&,
+//                      const std::optional<RHS_TYPE>&);
+// [31] bool operator>=(const optional<LHS_TYPE>&,
+//                      const std::optional<RHS_TYPE>&);
+// [31] bool operator> (const optional<LHS_TYPE>&,
+//                      const std::optional<RHS_TYPE>&);
+// [30] optional make_optional(bsl::allocator_arg_t,
+//                             allocator_type const&,
+//                             TYPE&&);
+// [30] optional make_optional(bsl::allocator_arg_t,
+//                             allocator_type const&,
```

Comment:
For consistency let's place it like in the rest of the code.

```suggestion
//                             const allocator_type&,
```

Reply:
fixed all instances